shard.yml

```
@@ -10,6 +10,7 @@ development_dependencies:
     github: crystal-ameba/ameba
     version: ~> 0.14.0
 
-crystal: 0.36.1
+# TODO: Update to "~> 1" once the time comes.
+crystal: ">= 0.36, < 2"
```

Upper boundary is redundant.

```suggestion
crystal: ">= 0.36"
```

---

Unless we have Crystal 2 with some breaking changes 🙂

---

You can say the same about changes in `1.1` and so on, so that's a moot argument.

---

Once there are changes which are not present in 1.0, this should be updated to `~> 1.1`. Until then, `>= 0.36, < 2` is fine. This is also followed by [crystal-mysql](https://github.com/crystal-lang/crystal-mysql/blob/master/shard.yml#L13), for example

---

Since the upper boundary is gonna cause the breakage anyway there's no point of adding it, causing false perception of compatibility.